igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/InfoController.groovy

```diff
@@ -71,6 +69,14 @@ class InfoController {
                     ]
                 }
             )
+            masterList.addAll(
```

Comment:
It might be a good idea to have an optional filter to restrict the list of masters. If someone only cares about jenkins, we shouldn't include travis master in the list

Reply:
I see what you mean but IMO this should be the subject of another ticket.